Gemfile.lock

```diff
@@ -393,7 +393,7 @@ GEM
     websocket-driver (0.6.4)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.2)
-    will_paginate (3.1.0)
+    will_paginate (3.1.5)
```

Comment:
Theres quite a few gems updated here, theres no reason for these to be updated as far as I can see and its usually a bad idea to update so many at the same time as it can cause unforeseen problems. Can you please reset the lock file to the version thats in master.